src/main/java/com/fusionflux/portalcubed/blocks/fizzler/AbstractFizzlerBlock.java

```diff
@@ -79,7 +78,6 @@ public VoxelShape getTriggerShape(BlockState state, BlockView world, BlockPos po
     }
 
     @Override
-    @SuppressWarnings("deprecation")
```

Comment:
Why'd you remove this?

Reply:
There's a deprecation warning in IntelliJ because you're overriding a deprecated method.